lib/Cro/HTTP/Router.pm6

```diff
@@ -947,15 +947,19 @@ module Cro::HTTP::Router {
 
         my %fallback = $mime-types // {};
         my $ext = $child eq '.' ?? $base.extension !! $child.IO.extension;
-        my $content-type = %mime{$ext} // %fallback{$ext} // 'application/octet-stream';
+
+        my sub get-mime($ext) {
+            %mime{$ext} // %fallback{$ext} // 'application/octet-stream';
+        }
+        my $content-type = get-mime($ext);
```

Comment:
I think it'd be neater to instead call `get-mime` in place of the single use of it below.